core/domain/exp_jobs_one_off.py

```diff
@@ -720,3 +720,55 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class TranslatorToVoiceArtistOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for migrating translator_ids to voice_artist_ids."""
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationRightsModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        translator_ids = item.translator_ids
+        if len(translator_ids) > 0:
+            exp_summary_model = exp_models.ExpSummaryModel.get_by_id(item.id)
+
+            if exp_summary_model is None or exp_summary_model.deleted:
+                item.voice_artist_ids = translator_ids
+                item.translator_ids = []
+                commit_message = 'Migrate from translator to voice artist'
+                commit_cmds = [{
+                    'cmd': 'change_role',
+                    'assignee_ids': translator_ids,
```

Comment:
Yes. This command also occurs in rights_manager.py and somewhere else in the codebase.

Reply:
Sorry for not being specific in the first place.

I'm talking about the `assignee_ids`, all the existing commands in the datastore only have `assignee_id`, right?

Reply:
I think for 'chang_role' command specifically, it also has 'new_role' and 'old_role'. We can see it in rights_manager.py. It's also specified in a docstring
`- 'change_role' (with assignee_id, new_role and old_role)`

Reply:
I agree on that, but none of the existing commands have `assignee_ids` (note `s` in the end)

Reply:
Oh right my bad. I'll fix that soon.

Reply:
In this case is it okay to pass in a list of translator_ids to 'assignee_id'? It works fine but it might seem a little weird. Or should I make multiple commits?

Reply:
I think commit_cmds is a list so we can have multiple commands in a single commit, right?